platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/network/NetworkPeerIdentifier.java

```diff
@@ -46,52 +45,49 @@ public class NetworkPeerIdentifier {
      */
     private final RateLimitedLogger noPeerFoundLogger;
 
-    private final RateLimitedLogger multiplePeersFoundLogger;
+    // a mapping of X500Principal and their peers
```

Comment:
Member docs should use /* */ comment style (only need be addressed in this PR if there are other needed changes)

Reply:
thank you. Will take care of it in a subsequent PR.